SEED-platform/seed
# feat(better): add results to original property state

#### Any background context you want to provide?
SEED is improving integration with BETTER

#### What's this PR do?
- makes request to get building analysis as JSON and stores it in the AnalsisPropertyView's parsed_results property
- adds a step to the BETTER pipeline to use the JSON results to update to _original_ PropertyState. Currently it stores the potential cost and energy savings.

#### How should this be manually tested?
Upload the file below. Import the required BETTER fields (City, State, Postal Code, Property Type) as well as the meters from that same file. Then Run the analysis for all or one of the properties. After it's completed, a new Column should be created for both of the columns mentioned above. Go to the column list profile settings, add those new BETTER columns (they both start with "BETTER") then view the inventory list and confirm the results were put into these columns.

[PM_BETTER_PORTFOLIO_INPUT.xlsx](https://github.com/SEED-platform/seed/files/6871487/PM_BETTER_PORTFOLIO_INPUT.xlsx)


#### What are the relevant tickets?
#2770 

#### Screenshots (if appropriate)
![Screen Shot 2021-07-23 at 4 14 49 PM](https://user-images.githubusercontent.com/18518728/126846491-8e016f22-7c3f-48a8-b72c-37b7ee17b08e.png)
